util/config/validators/validate_access.py

```diff
@@ -15,11 +15,12 @@ def validate(cls, validator_context):
             # Make sure we have at least one OIDC enabled.
             github_login = config.get("FEATURE_GITHUB_LOGIN", False)
             google_login = config.get("FEATURE_GOOGLE_LOGIN", False)
+            openshift_login = config.get("FEATURE_OPENSHIFT_LOGIN", False)
 
             login_manager = OAuthLoginManager(config, client=client)
             custom_oidc = [s for s in login_manager.services if isinstance(s, OIDCLoginService)]
 
-            if not github_login and not google_login and not custom_oidc:
+            if not github_login and not google_login and not openshift_login and not custom_oidc:
```

Comment:
Now that this list is growing, we could use 
```python
if not any([github_login, google_login, openshift_login]) and not custom_oidc:
```

Reply:
yep agreed, doesnt read very well for a giant list of and/or conditions

Reply:
Fixed not pushed